cookiecutter/generate.py

```diff
@@ -163,6 +167,11 @@ def render_and_create_dir(dirname, context, output_dir):
     dir_to_create = os.path.normpath(
         os.path.join(output_dir, rendered_dirname)
     )
+    if os.path.exists(dir_to_create):
+        if fail_if_exists:
+            msg = 'Error: "{0}" directory already exists'.format(dir_to_create)
```

Comment:
Feel free to omit the index in the in error message. We dropped Py26 support a while ago. :smile_cat: